controllers/whoToFollow/whoToFollowV2.js

```diff
@@ -54,7 +54,7 @@ module.exports = async (req, res) => {
                             a.is_anonymous = false
                             AND a.is_banned = false
                             AND a.blocked_by_admin = false
-                            AND a.karma_score > 30
+                            AND a.karma_score > 45
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the karma score threshold from 30 to 45 might exclude users who were previously included in the results. Ensure that this change aligns with your application's requirements and user expectations.
```diff
- AND a.karma_score > 30
+ AND a.karma_score > 45
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->